shopgate/pwa
# Refactored usage of Suspense component

# Description

This ticket is about to refactor the usage of the `Suspense` component. Before the whole content was wrapped by the component, which caused blank pages and re-renderings when any component (route, widget, portal) was lazy loaded. This caused display errors of e.g. product sliders.

Now every lazy loaded component is wrapped by `Suspense` and has its own loading indicator state.

## Type of change


## How to test
For me it was hard do reproduce the behaviour within the dev environment. But there are some release candidates which can be tried out within sandbox shops. If you configure a product slider right at the top of the startpage, you can easily reproduce the slider issue.

- 6.11.0-rc.5 page renders / white page / page re-renders => product slider is broken
- 6.11.0-rc.6 same behaviour like rc.5, but you'll see a red square when the global Suspend component falls back
- 6.11.0-rc.7 based on this branch without the white page and the broken product slider
